source/gx/tilix/terminal/terminal.d

```diff
@@ -1044,6 +1044,34 @@ private:
         vteHandlers ~= vte.addOnKeyPress(delegate(Event event, Widget widget) {
             if (vte is null) return false;
 
+            // If control-backspace is registered, convert it to control-*, which is
+            // used interchangeably in some terminal emulators,
+            // often for removing a word instead of a character.
+            if (gsProfile.getString(SETTINGS_PROFILE_CTRL_BACKSPACE_KEY) != SETTINGS_PROFILE_CTRL_BACKSPACE_VALUES[0] && (event.key.keyval == GdkKeysyms.GDK_BackSpace) && (event.key.state & ModifierType.CONTROL_MASK)) {
+                switch (gsProfile.getString(SETTINGS_PROFILE_CTRL_BACKSPACE_KEY)) {
+                    case SETTINGS_PROFILE_CTRL_BACKSPACE_VALUES[1]: // ^w
+                        vte.feedChild("\u0017");
+                        return true;
+                    case SETTINGS_PROFILE_CTRL_BACKSPACE_VALUES[2]: // ^h
+                        vte.feedChild("\u0008");
+                        return true;
+                    case SETTINGS_PROFILE_CTRL_BACKSPACE_VALUES[3]: // ^?
+                        vte.feedChild("\u001F");
+                        return true;
+                    case SETTINGS_PROFILE_CTRL_BACKSPACE_VALUES[4]: // DEL
+                        vte.feedChild("\u0007");
```

Comment:
`\u0007` is not `DEL`. `\u0007` is `BEL`. I've never seen a terminal sending `BEL` for a keypress of Ctrl-Backspace.